lib/config/rules/functionality.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
   // Require a `default` case in all `switch` statements.
   'switch-default': false,
   // Requires `===` and `!==` in place of `==` and `!=`.
-  'triple-equals': true,
+  'triple-equals': [true, 'allow-null-check'],
```

Comment:
I actually meant to create branches from that current branches because there are dependent changes. Eg. `commitizen` commit messages. 

Where the ugliness started was from the initial [Branch Compare](https://github.com/Shopify/tslint-config-shopify/pull/12/files/0436610c94f91be3ea66a6b7abb2d3ba3c543369#diff-04c6e90faac2675aa89e2176d2eec7d8).